base/io.jl

```diff
@@ -84,7 +84,20 @@ Read a single value of type `T` from `stream`, in canonical binary representatio
 
     read(stream::IO, String)
 
-Read the entirety of `stream`, as a String.
+Read the entirety of `stream`, as a `String`.
```

Comment:
I like when the function signature and examples match, perhaps change `` `stream` `` to `` `io` `` here and in the two signatures? Seems like `io` is used in almost all signatures.